code/game/objects/items/storage/boxes.dm

```diff
@@ -530,6 +530,13 @@
 	max_combined_w_class = 21
 	use_to_pickup = 1 // for picking up broken bulbs, not that most people will try
 
+/obj/item/storage/box/lights/attackby(obj/item/I, mob/user, params)

```

Comment:
You're strictly defining this for the `box/lights` subtype, which doesn't look like it's different from a regular box, so that's going to be confusing. Bad design! Move this proc to a regular `/obj/item/storage/box`. I'd also add a check to make sure it's a cyborg making this thing happen, because a human trying to put a light replacer into the bulb box might happen.

`if(issilicon(I.loc))` <- cyborg/AI check, though an AI is unlikely to wield anything.